R/combine_distance_matrices.R

```diff
@@ -0,0 +1,56 @@
+# combine_distance_matrices.R
+
+#' Combine Distance Matrices
+#'
+#' \code{combine_distance_matrices} This is a helper function for \code{\link{clusterSystems}} and is unlikely to be useful elsewhere.
+#'
+#' @param mode Method by which to combine distance matrices
+#' @param distanceMatrices List of objects of class distance.
+#'
+#' @return single object of class distance.
+#'
+#' @author \href{https://orcid.org/0000-0001-5724-2252}{Rachel Silverstein} (aut)
+#'
+#'
+combine_distance_matrices <- function(mode, distanceMatrices) {
+  if (length(distanceMatrices) > 1) {
+
+    if (mode == 'sum') {
```

Comment:
I would put the check for the mode type inside the else condition (where i != 1) to simplify these control statements

Reply:
I don't think it can go there. The structure is "if length >1 AND mode == x" then do something... else do something else.

Reply:
I mean you can put the check for the mode type inside the for loop (under the condition where i != 1). That way you don't need to repeat the code for the for loop and i == 1 condition.

Reply:
Oh yes, you are right. Fixing it.